etc/c/zlib.d

```diff
@@ -566,14 +567,13 @@ int deflateInit2(z_streamp strm,
 }
 /*
      This is another version of deflateInit with more compression options.  The
-   fields next_in, zalloc, zfree and opaque must be initialized before by the
-   caller.
+   fields zalloc, zfree and opaque must be initialized before by the caller.
 
      The method parameter is the compression method.  It must be Z_DEFLATED in
    this version of the library.
 
      The windowBits parameter is the base two logarithm of the window size
-   (the size of the history buffer).  It should be in the range 8 .. 15 for this
+   (the size of the history buffer).  It should be in the range 8..15 for this
```

Comment:
The style checker will always fire for this, so better revert to the previous range style.

Reply:
OK, will do.

Reply:
Done.